### PR TITLE
[3.x] Fix / Only remove cookie fields when removing sensitive data from logs

### PR DESCRIPTION
**Pull request by valentino-amadeus**
#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/utils/log.ts [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
+const SENSITIVE_FIELDS = ['cookie', 'Cookie', 'vtexIdclientautcookie', 'error']
 
-    SECOND_LEVEL_SENSITIVE_FIELDS.forEach(field => {
-        if (field[0] in log) {
-            delete log[field[0]][field[1]]
-        }
-    })
-}
\ No newline at end of file
+export const cleanLog = (log: any) => {
```
Comment (gris) [inline src/utils/log.ts:5]: Why do we have this mismatch between what is expected between `log` and `json` in the function `cleanJson`?

Comment (valentino-amadeus) [reply src/utils/log.ts]: We don't anymore :jajaja:. Thanks!

File: src/utils/json.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    for (const key of Object.keys(json)) {
+        let deleted = false
+        for (const field of targetFields) {
+            if (key === field) {
+                delete json[key]
+                deleted = true
+            }
+        }
+
+        if (!deleted && typeof json[key] === 'object') {
```
Comment (brunoabreu) [inline src/utils/json.ts:11]: Same issue mentioned [here](https://github.com/vtex/node-vtex-api/pull/494#discussion_r821669949)

**Review by brunoabreu — CHANGES_REQUESTED** [review]

File: src/utils/json.ts (hunk 2 above)
Comment (valentino-amadeus) [reply src/utils/json.ts]: [Done](https://github.com/vtex/node-vtex-api/pull/495/commits/d4df050618492c97991da4f78c5842329df1d153). Thanks!

**Review by brunoabreu — APPROVED** [review]

**Pull request merged**